server/bin/unittests

```diff
@@ -111,7 +111,7 @@ function _local_find {
 
 function _normalize_output {
     # Fix up tmp directory references
-    sed 's;tmp/pbench-\([-a-zA-Z0-9]*\)\.[0-9][0-9]*/;tmp/pbench-\1.NNNN/;' $*
```

Comment:
What required this change to the regular expression?

Reply:
still updating the PR for unit tests. Will ping you when it is ready for review.

Reply:
Still wondering why we needed to change this regular expression.  Can you add a comment here and in the code if necessary?